File: db/jsonToDB.js
const sqlite = require("sqlite3").verbose();
const util = require("util");
const q = require("./quotes.json")["Thomas Jefferson"];
// Quotes(quote text not null, id int not null primary key, source text not null, usage int not null)
let db = new sqlite.Database('./quotes.db', sqlite.OPEN_READWRITE, (err) => {
    if (err) {
        console.error(err.message);
    }
    console.log('Connected to the database.');
});
let count = 10;
db.run = util.promisify(db.run);
q.forEach(async key => {
    count++;
    await db.run(`insert into Quotes(quote, id, source, usage) values('${key}', ${count}, 'Thomas Jefferson', 0)`);
})
/* for (const key in q) {
    count++;
    db.run(`insert into Quotes(quote, id, source, usage) values('${key}', ${count}, 'Thomas Jefferson', 0)`);
} */
db.close((err) => {
    if (err) {
        console.error(err.message);
    }
    console.log('Close the database connection.');
});

File: db/jsonToDb.py
#!/usr/bin/env python3
import sqlite3
import json
data = json.load(open("quotes.json"))
db = sqlite3.connect("quotes2.db")
source = "Scarlet Pimpernel by Baroness Orczy"
for count, q in enumerate(data[source], start=68):
    #db.cursor().execute("insert into Quotes(quote, id, source, usage) values('" +q+"}', "+str(count)+", 'George Washington', 0)")
    db.cursor().execute("insert into Quotes(quote, id, source, usage) values(?,?,?,?)",(q, count, source, 0))
#print(data["Thomas Jefferson"])
db.commit()
db.close()